tests/unit/test-helpers.js

```diff
@@ -0,0 +1,19 @@
+import { shallowMount, mount } from '@vue/test-utils';
+import applyPrototypes from '@/helpers/applyPrototypes.js';
+
+export default {
+  shallowMount: function (component, options) {
+    options = options || {};
+    const wrapper = shallowMount(component, options);
+    console.log(0, { component });
+    applyPrototypes(wrapper)
+    return wrapper;
+  },
+  mount: function (component, options) {
+    options = options || {};
+    const wrapper = mount(component, options);
+    console.log(1, { component });
+    applyPrototypes(wrapper)
+    return wrapper;
```

Comment:
Here are the actual `mount` and `shallowMount` functions used by the tests. They pull in the actual `mount`/`shallowMount` from `@vue/test-utils`. I've verified the installed version of VTU is `2.0.0-rc.6` (according to `node_modules/@vue/test-utils/package.json`).

When the `component` variable passed in is logged, it looks like this:
```js
{
  name: 'HelloWorld',
  components: { FsExample: [Object], LinkList: [Object] },
  props: { msg: [Object] },
  data: [Function: data],
  methods: { toggleDevTools: [Function: toggleDevTools] },
  computed: {
    devMode: [Function: devMode],
    versions: [Function: versions],
    vueVersion: [Function: vueVersion]
  },
  render: [Function: render]
}
```
but after passing it into VTU's `mount` or `shallowMount` it comes out as `undefined`.

Reply:
when running `npm t`, most of the errors look like this now:

![err](https://user-images.githubusercontent.com/4629794/122936408-21e52d80-d33f-11eb-8c45-a03be28aa94c.png)